lava-v2-jobs-from-api.py

```diff
@@ -335,6 +337,8 @@ def main(args):
                         help="type of dataset to receive in callback")
     parser.add_argument("--defconfigs", default=0,
                         help="Expected number of defconfigs from the API")
+    parser.add_argument("--image-name", default="boot.img", dest='boot_image',
```

Comment:
Can't we just drop this parameter and call the image `boot.img` in the fastboot template directly?  Alternatively, this could be set as a DeviceType parameter in YAML.

Reply:
@khilman suggested that should be a parameter, but if you want to make it a static value it would be easy to do that.

Reply:
I think it's useful to be able to set this as a parameter, but not on the command line for this script which generates multiple jobs.  So this could be a template parameter coming from the YAML test config.

Reply:
I ve droped this commit.